### ISSUE TITLE
Upgrade to apache jasper 8.5.24

### ISSUE THREAD

**Issue by janbartel** [eclipse/jetty.project#2079]
Apache Jasper 8.5.24 is the first version to fully support jdk 9.  Even though Jetty already supports jdk9 enough to make jsp work, there are some cases where the supporting jetty classes are not being used - see for example https://github.com/jetty-project/jasper-jsp/issues/3

**Comment by janbartel** [comment]
Opened CQ 15290 at eclipse.

**Comment by cemo** [comment]
@janbartel Is CQ issues are publicly available? Do you have ETA for release?

**Comment by janbartel** [comment]
@cemo  I think you have to be an Eclipse committer to see them AFAIK. No ETA on an approval from Eclipse as yet. I would imagine we are at least 3 weeks from doing another 9.4 release.

**Comment by joakime** [comment]
@cemo the main page for CQ's is IP Zilla at https://dev.eclipse.org/ipzilla/

The direct CQ link (at least for us that are logged in) is https://dev.eclipse.org/ipzilla/show_bug.cgi?id=15290

**Comment by janbartel** [comment]
@cemo I just released the org.mortbay.jasper 8.5.24 jars.  Do you want to build jetty locally with them and test your usecase?

**Comment by cemo** [comment]
@janbartel I tested and working properly. I just added `apache-jsp` with existing release `9.4.8.v20171121` and I saw that our pages is rendered properly. Also the [demo] project is working properly. 

 We will investigate very detailed next week. In case an error, I will let you now details as well. 

```
    <dependency>
      <groupId>org.mortbay.jasper</groupId>
      <artifactId>apache-jsp</artifactId>
      <version>${apache-jsp.version}</version>
    </dependency>
```

[demo]: https://github.com/cemo/bug-jetty-jsp
